pinot-tools/src/main/java/org/apache/pinot/tools/streams/MeetupRsvpJsonStream.java

```diff
@@ -38,20 +36,17 @@ public MeetupRsvpJsonStream(boolean partitionByKey)
   }
 
   @Override
-  protected MessageHandler.Whole<String> getMessageHandler() {
+  protected Consumer<RSVP> createConsumer() {
```

Comment:
For the json stream, we actually expect the event to have nested json inside. With the current approach, all the fields are flattened

Reply:
Unfortunately, there's no record of what that JSON looked like. I'm trying to stop the CI build from failing so people can merge PRs.